paper/src/main/java/com/denizenscript/denizen/paper/events/BlockPreDispenseScriptEvent.java

```diff
@@ -0,0 +1,71 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import io.papermc.paper.event.block.BlockPreDispenseEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class BlockPreDispenseScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // <block> tries to dispense <item>
+    //
+    // @Group Block
+    //
+    // @Location true
+    //
+    // @Cancellable true
+    //
+    // @Triggers before a block dispenses an item.
```

Comment:
There should be an explanation here of when/why you would use this event vs the existing `<block> dispenses <item>`